<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>UART Watcher</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QPlainTextEdit" name="plainTextEdit_raw_orig"/>
          <widget class="QPlainTextEdit" name="plainTextEdit_raw_hex"/>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lineEdit_Send"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_send">
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox_hex">
            <property name="text">
             <string>Hex</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Packet Sender</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton_demo_1">
            <property name="text">
             <string>DEMO1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_demo_2">
            <property name="text">
             <string>DEMO2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_demo_3">
            <property name="text">
             <string>DEMO3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_demo_4">
            <property name="text">
             <string>DEMO4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_demo_5">
            <property name="text">
             <string>DEMO5</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_demo_6">
            <property name="text">
             <string>DEMO6</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_demo_7">
            <property name="text">
             <string>DEMO7</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_demo_8">
            <property name="text">
             <string>DEMO8</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>Set Input Variable</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Get Output Variable</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Set Function Block Diagram</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Get Input Block List</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Get Output Block List</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Get Function Block List</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Start Function Block Diagram</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Stop Function Block Diagram</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_target">
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="displayIntegerBase">
             <number>16</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_send_cmd">
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_send_payload"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox">
            <property name="minimum">
             <double>-100.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_to_hex"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>819</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionProtocol_Connect"/>
   <addaction name="actionClearLog"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icon/unlink-symbol (1).png</normaloff>
     <normalon>:/icons/icon/link-symbol (3).png</normalon>
     <disabledon>:/icons/icon/link-green.png</disabledon>:/icons/icon/unlink-symbol (1).png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Click to setup a connection</string>
   </property>
  </action>
  <action name="actionProtocol_Connect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icon/pause.png</normaloff>
     <normalon>:/icons/icon/play-button.png</normalon>:/icons/icon/pause.png</iconset>
   </property>
   <property name="text">
    <string>Protocol Connect</string>
   </property>
   <property name="toolTip">
    <string>Protocol Connect</string>
   </property>
  </action>
  <action name="actionClearLog">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icon/garbage1.png</normaloff>:/icons/icon/garbage1.png</iconset>
   </property>
   <property name="text">
    <string>ClearLog</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
